optaplanner-persistence/optaplanner-persistence-jackson/src/test/java/org/optaplanner/persistence/jackson/api/OptaPlannerJacksonModuleTest.java

```diff
@@ -29,6 +30,7 @@
     @Test
     public void polymorphicScore() {
         ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.enable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY);
```

Comment:
@contextshuffling Would you mind adding a comment here, describing why this line is necessary?